packages/twenty-apps/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "twenty-apps",
+  "version": "0.1.0",
+  "description": "Collection of Twenty CRM applications and extensions",
+  "private": true,
+  "keywords": [
+    "twenty",
+    "crm",
+    "applications",
+    "extensions",
+    "marketplace"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/twentyhq/twenty.git",
+    "directory": "packages/twenty-apps"
+  },
+  "license": "AGPL-3.0",
+  "scripts": {
+    "lint": "echo 'No linting needed for application manifests'",
+    "validate": "ts-node scripts/validate-apps.ts",
```

Comment:
Remove validate for now

Let's add another very small jest test since we remove validate-apps.test.ts